deployability/modules/jobflow/tests/conftest.py

```diff
@@ -62,11 +62,11 @@ def w_processor(request) -> WorkflowProcessor:
     threads = request.param.get('threads', 1)
     log_level = request.param.get('log_level', 'info')
     schema_file = request.param.get('schema_file', 'schema.yaml')
-    with patch("workflow_engine.workflow_processor.WorkflowFile") as file_mock:
+    with patch("jobflow.workflow_processor.WorkflowFile") as file_mock:
```

Comment:
Shouldn't the name of the workflow_processor method and WorkflowFilw be changed?

Reply:
Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8